packages/client/modules/userDashboard/components/OrgTeamMembers/OrgTeamMembers.tsx

```diff
@@ -64,17 +66,19 @@ export const OrgTeamMembers = (props: Props) => {
           </Link>
         </Button>
         <h1 className='flex-1 text-2xl leading-7 font-semibold'>{team.name}</h1>
-        <div className='ml-auto'>
-          {showMenuButton && (
-            <Button
-              shape='circle'
-              variant='ghost'
-              onClick={togglePortal}
-              ref={originRef}
-              className='bg-slate-400'
-            >
-              <MoreVert />
-            </Button>
+        <div className='ml-auto flex items-center'>
+          <InviteTeamMemberAvatar teamId={team.id} teamMembers={teamMembers} />
+          {showDeleteButton && (
+            <div className='group mx-1.5 cursor-pointer'>
+              <div className='flex h-7 justify-center'>
+                <span className='h-6 w-6 self-center text-tomato-500 group-hover:text-tomato-600'>
```

Comment:
+1 i think this should be a gray button. I don't think we have a tomato-500 delete icon elsewhere in the app